ldap/servers/slapd/back-ldbm/ldbm_config.h

```diff
@@ -104,6 +104,9 @@ struct config_info
 #define CONFIG_DB_VERBOSE "nsslapd-db-verbose"
 #define CONFIG_DB_DEBUG "nsslapd-db-debug"
 #define CONFIG_DB_LOCK "nsslapd-db-locks"
+#define CONFIG_DB_LOCKS_MONITORING "nsslapd-db-locks-monitoring"
```

Comment:
Being a on/off toggle it could also be nsslapd-db-locks-monitoring-enable ?

Reply:
I've created it this way because we already have **nsslapd-disk-monitoring** with the same functionality so I've decided to keep it consistent.
I don't have a strong opinion on this so if you think it's better as **nsslapd-db-locks-monitoring-enable**, - I can change it.

Reply:
I tend to prefer nsslapd-db-locks-monitoring-enable, but it also depends the amount of work (code, test, design) that costs. If it is painful then ignore my (late) comment

Reply:
Sure, sounds good!